applications/portfoliomanager/src/portfoliomanager/server.py

```
@@ -43,7 +39,7 @@
     cache_logger_on_first_use=True,
 )
 
-from .enums import PositionAction, PositionSide, TradeSide  # noqa: E402
+from .enums import PositionSide, TradeSide  # noqa: E402
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)